book/intro.md

```diff
@@ -76,7 +76,10 @@ Reth implements the specification of Ethereum as defined in the [ethereum/execut
 1. We operate multiple nodes at the tip of Ethereum mainnet and various testnets.
 1. We extensively unit test, fuzz test and document all our code, while also restricting PRs with aggressive lint rules.
 
-We intend to also audit / fuzz the EVM & parts of the codebase. Please reach out if you're interested in collaborating on securing this codebase.
+We have completed an audit of the [Reth v1.0.0-rc.2](https://github.com/paradigmxyz/reth/releases/tag/v1.0.0-rc.2) with [Sigma Prime](https://sigmaprime.io/), the developers of [Lighthouse](https://github.com/sigp/lighthouse), the Rust Consensus Layer implementation. Find it [audit/sigma_prime_audit_v1.pdf](https://github.com/paradigmxyz/reth/blob/main/audit/sigma_prime_audit_v1.pdf).
```

Comment:
please make it a permalink instead, in case we change the location of this PDF in the repo

Reply:
changed, please take another look.